packages/bierzo-wallet/src/store/extension/index.unit.spec.ts

```diff
@@ -44,10 +43,14 @@ withChainsDescribe("Extension reducer", () => {
     const extension = await getExtensionStatus();
     store.dispatch(setExtensionStateAction(extension.connected, extension.installed, extension.identities));
 
+    const extensionIdentities: { [chain: string]: Identity } = {
+      "ethereum-eip155-5777": identitiesResponse[0],
+    };
+
     const extensionState = store.getState().extension;
     expect(extensionState.connected).toBeTruthy();
     expect(extensionState.installed).toBeTruthy();
-    expect(extensionState.identities).toBe(identitiesResponse);
+    expect(extensionState.identities).toStrictEqual(extensionIdentities);
```

Comment:
Interesting new matcher `toStrictEqual`. I think a simple `toEqual` would do the job as well